resources/config_bundles/macos/patch_order.list

```diff
@@ -4,4 +4,5 @@ ungoogled-chromium/macos/fix-gn-bootstrap.patch
 ungoogled-chromium/macos/fix-gn-safe_browsing.patch
 ungoogled-chromium/macos/fix-mapped_file.patch
 ungoogled-chromium/macos/fix-visibility.patch
-ungoogled-chromium/macos/macos-fix-building-without-safebrowsing.patch
+ungoogled-chromium/macos/fix-building-without-safebrowsing.patch
```

Comment:
I'm thinking it may be better to move this to the `common` bundle because these changes may be nice for other platforms.

Reply:
ok